ci/jenkins/jobs/job-templates.yaml

```diff
@@ -46,3 +46,42 @@
         triggered-status: '{triggered_status}'
         started-status: '{started_status}'
     wrappers: '{wrappers}'
+
+- job-template:
+      name: '{test_name}-daily'
```

Comment:
dont have daily.. we can move this job to weekly too..just test name is good enough

Reply:
Updated it.